src/main/java/core/basesyntax/sevice/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.sevice.impl;
+
+import core.basesyntax.service.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> readFromFile(String fileName) {
+        try {
+            return Files.readAllLines(Path.of(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t find the file " + fileName, e);

```

Comment:
The error message should be more informative. It's better to specify that the file cannot be read, not just that it can't be found, as the issue might be related to reading the file, not just its existence.